server/src/utils/getConfig.ts

```diff
@@ -86,7 +89,10 @@ export const getServerConfig = (): ServerConfigModel => {
     waitingTitle: process.env[VV_WAITING_TITLE_ENV_NAME] ?? defaultConfig.waitingTitle,
     waitingSubtitle: process.env[VV_WAITING_SUBTITLE_ENV_NAME] ?? defaultConfig.waitingSubtitle,
     logoUrl: process.env[VV_LOGO_URL_ENV_NAME] ?? defaultConfig.logoUrl,
-    postCall: getPostCallConfig(defaultConfig)
+    postCall: getPostCallConfig(defaultConfig),
+    cosmosDBConnectionString: process.env[VV_COSMOS_DB_CONNECTION_STRING],
+    cosmosDBEndpoint: process.env[VV_COSMOS_DB_ENDPOINT],
+    cosmosDBName: process.env[VV_COSMOS_DB_NAME]
```

Comment:
The cosmosDB configs should also be fetched from the defaultConfig

`process.env[VV_COSMOS_DB_NAME] ?? defaultConfig.cosmosDBName`

Reply:
If we planned to include the db configs in `defaultConfig`, then we can grab it from it. With current implementation, nothing was from the `defaultConfig`.